### ISSUE TITLE
L2 EVMs launch overview

### ISSUE THREAD

**Issue by tomasklim** [trezor/trezor-suite#15478]
Work needs to be done before we can introduce `Base`, `Optimism`, `Arbitrum`  and other `L2s`

```[tasklist]
### 2025.1.1 - experimental features
- [ ] https://github.com/trezor/trezor-suite/issues/14725
- [ ] https://github.com/trezor/trezor-suite/issues/14724
- [ ] #15818
- [ ] #15821
- [ ] #15817
- [ ] https://github.com/trezor/trezor-suite/issues/15896
- [ ] https://github.com/trezor/trezor-suite/issues/16084
- [ ] https://github.com/trezor/trezor-suite/issues/15890
- [ ] #15816
- [ ] https://github.com/trezor/trezor-suite/issues/16121
- [ ] https://github.com/trezor/trezor-suite/issues/16166
- [ ] https://github.com/trezor/trezor-suite/issues/16162
- [ ] https://github.com/trezor/trezor-suite/issues/16193
- [ ] https://github.com/trezor/trezor-suite/issues/16200
```

```[tasklist]
### Prerequisites for launch
- [ ] https://github.com/trezor/trezor-suite/issues/16164
- [ ] #15820
- [ ] https://github.com/trezor/trezor-suite/issues/16163
- [ ] https://github.com/trezor/trezor-suite/issues/16161
- [ ] #15819
- [ ] https://github.com/trezor/trezor-suite/issues/16165
- [ ] https://github.com/trezor/trezor-suite/issues/15546
- [ ] https://github.com/trezor/trezor-suite/issues/16773
```

```[tasklist]
### Nice-to-have it
- [ ] https://github.com/trezor/trezor-suite/issues/15215
- [ ] https://github.com/trezor/trezor-suite/issues/16194
- [ ] #14673
- [ ] #15417
- [ ] https://github.com/trezor/trezor-firmware/issues/3994
- [ ] #15822
- [ ] https://github.com/trezor/trezor-suite/issues/8037
```
